packages/react-native-web/src/exports/StatusBar/index.js

```diff
@@ -7,16 +7,92 @@
  * @providesModule StatusBar
  * @flow
  */
-
 import { Component } from 'react';
 
+
+const {head} = document
+
+let _barStyle = 'default'
+let _hidden = false
+let _translucent = false
+
+
+function getMetaTag(attrName) {
+  const htmlCollection = head.getElementsByTagName('meta')
```

Comment:
No need to collect all the meta tags when all you need is something like 

```
document.querySelector(`meta[name=${attrName}]`)
```

Reply:
Good trick, thank you! :-) Fixed.